web/packages/extension/tools/.eslintrc.yaml

```diff
@@ -1,4 +0,0 @@
-env:
-  node: true
-parserOptions:
-  ecmaVersion: 2022 # Needed for top-level-await.
```

Comment:
Umm, is this present in the flat config? I'm only seeing `packages/core/tools/**` there.

Reply:
Good catch. It's defaulting to "`latest`" which is newer.

We *should* switch to `"latest"` but didn't want to make more changes than needed from base config.

Reply:
Fixed! It should have been extension, not core